docs/app/Examples/collections/Form/Validation/FormValidatingOnBlurAndOtherEventsExample.js

```diff
@@ -0,0 +1,73 @@
+import React, {Component} from 'react';
+import {Button, Checkbox, Field, Fields, Form, Input, Message} from 'stardust';
+
+export default class FormValidatingOnBlurAndOtherEventsExample extends Component {
```

Comment:
Another example, this time showing inline form validation on blur.  This uses Semantic UI popups with framework animations.  They are displayed on blur if the field is failing validation.  This was added from [this](http://semantic-ui.com/behaviors/form.html#validating-on-blur-and-other-events) Semantic UI example.